www/dart-sass/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dart-sass
+version             1.32.8
+categories          www textproc
+platforms           darwin
+license             MIT
+maintainers         @queensferryme openmaintainer
+description         The reference implementation of Sass, written in Dart.
+long_description    ${description}
+homepage            https://sass-lang.com/dart-sass
+
+fetch.type          git
+git.url             https://github.com/sass/dart-sass.git
+git.branch          ad4a1692a8166a16fc57fc6f16fc5f0b15dba165
```

Comment:
```tcl
PortGroup           github 1.0

github.setup        sass dart-sass 1.32.8
```

I recommend looking at the [GitHub PG](https://guide.macports.org/#reference.portgroup.github), which can help simplify things. This also deals with the `name` and `version`, which can then be removed.

Reply:
fixed, thx for the advice.